samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -36,7 +36,7 @@ static struct bt_iso_chan_ops iso_ops = {
 };
 
 static struct bt_iso_chan_io_qos iso_tx_qos = {
-	.sdu = 502, /* bytes */
+	.sdu = CONFIG_BT_ISO_TX_MTU, /* bytes */
```

Comment:
502 byte SDU size changed to 251 byte, this will change on-air behavior in terms of PDUs transmitted and ISO interval used.

Reply:
The SDU we set cannot be higher than `CONFIG_BT_ISO_TX_MTU`. 

Currently, as we haven't implemented fragmentation for it yet, we shouldn't set the CONFIG_BT_ISO_TX_MTU higher than 251.

Reply:
Then the Kconfig range need to be fixed to maximum 251, no point in allowing an application select 4095 as MTU size.

Reply:
I agree that the Kconfig needs an update. 

This needs additional testing to validate whether or not fragmentation is properly supported. The minimum value of 23 in the Kconfig also doesn't make sense, as minimum broadcast SDU can be 1, and minimum connected SDU can be 0. 

Will test and update in a different PR.

Reply:
```suggestion
	.sdu = sizeof(uint32_t), /* bytes */
```